test/transmuxer.test.js

```diff
@@ -2074,6 +2074,45 @@ QUnit.test('calculates baseMediaDecodeTime values from the first DTS ever seen a
   QUnit.equal(tfdt.baseMediaDecodeTime, 90, 'calculated baseMediaDecodeTime');
 });
 
+QUnit.test('doesn\'t adjust baseMediaDecodeType when configured to', function() {
+  videoSegmentStream.setOptionsForTesting_({keepOriginalTimestamps: true});
```

Comment:
I think it would be better if we instead updated the QUnit `setup` function for this module (line 1425) to add an options object and then alter that object here instead of adding an api for testing to the actual source.
e.g.
```
var track = {};
var options = {};
videoSegmentStream = new VideoSegmentStream(track, options);
videoSegmentStream.options = options;
videoSegmentStream.track = track;
.....
```
and then here just 
```
videoSegmentStream.options.keepOriginalTimestamps = true;
```

Reply:
Done